tests/integration/utils.sh

```diff
@@ -167,6 +167,43 @@ teardown_container_environment()
   done
 }
 
+# completely remove the container environment: the containers and the images
+function teardown_container_environment()
+{
+  local container_name
+  local distro
+  local img
+
+  for distro in "${DISTROS[@]}"; do
+    container_name="kw-${distro}"
+    podman container exists "${container_name}"
+    if [[ "$?" -eq 0 ]]; then
+      say "Removing podman container kw-${distro}."
+      teardown_single_container "${container_name}"
+    fi
+
+    img="kw-${distro}"
+    podman image exists "${img}"
+    if [[ "$?" -eq 0 ]]; then
+      say "Removing podman ${img} image."
+      podman image rm "${img}"
+      if [[ "$?" -ne 0 ]]; then
+        complain "Failed to remove ${img} image."
+      fi
+    fi
+
+    img="docker.io/library/${distro}"
```

Comment:
I think we should make this docker prefix path a global read-only variable because, in the future, if we want to use something different, we can just change the global variable.

Reply:
True, indeed